tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -118,6 +118,8 @@ class SbtServerSuite
              |object A {
              |  val foo: Int = "aaa"
              |}
+             |/.metals/
```

Comment:
Added so the workspace is detected as a metals project. Normally opening `.scala` file should be enough.

Reply:
Why wouldn't it be detected as a metals project though here? We do have a scala file inside the workspace

Reply:
Yeah, but we only look for specific paths, like `src/...` or `sth/src/...` to avoid searching the whole workspace. Generally, doing `didOpen` should also have solved this, but then I can't use `initialize`  "as is".

Reply:
don't we create build.sbt etc here?

Reply:
Yes, but again, we only specific paths and a top-level `build.sbt` would be detected, but here it will be in `inner/build.sbt`. I can change the logic to always resolve workspace folder as a metals project if there is a build tool found.

Reply:
ach, right, makes sense. Let's leave it as is